pkg/hub/store/store_test.go

```diff
@@ -2,27 +2,27 @@ package store
 
 import (
 	"database/sql"
+	"testing"
+
 	applicationv1 "github.com/kobsio/kobs/pkg/kube/apis/application/v1"
 	dashboardv1 "github.com/kobsio/kobs/pkg/kube/apis/dashboard/v1"
 	teamv1 "github.com/kobsio/kobs/pkg/kube/apis/team/v1"
 	userv1 "github.com/kobsio/kobs/pkg/kube/apis/user/v1"
 	"github.com/stretchr/testify/require"
-	"testing"
 )
 
 func TestStore(t *testing.T) {
```

Comment:
Please add a test for `SavePlugins` and `SaveClusters`

Reply:
Missing tests are added